iiop-openjdk/src/main/resources/org/wildfly/iiop/openjdk/LocalDescriptions.properties

```diff
@@ -26,6 +26,7 @@ iiop-openjdk.export-corbaloc=Indicates whether the root context should be export
 # security configuration properties.
 iiop-openjdk.support-ssl=Indicates whether SSL is to be supported (on) or not (off).
 iiop-openjdk.security-domain=The name of the security domain that holds the key and trust stores that will be used to establish SSL connections.
+iiop-openjdk.security-domain.deprecated=The name of the security domain that holds the key and trust stores that will be used to establish SSL connections.
```

Comment:
This message should be a description of the deprecation, similar to what follows an @deprecated in Javadoc.

Perhaps:

"Legacy security domains are not supported in a running server. Only usable in a domain configuration to support secondary hosts running earlier versions"

Reply:
@bstansberry thanks, fixed